index.js

```diff
@@ -182,9 +182,9 @@ Plotly.prototype.getFigure = function (fileOwner, fileId, callback) {
             } catch (e) {
                 callback(e);
             }
-            
-            if (body.error) {
-                callback(body.error);
+
+            if (body.error || !body.payload) {
+                callback(body.error || 'Something went wrong.');
```

Comment:
If the response from the server is an html doc, body is a string and does not have a `payload` child object so line 191 below will throw an uncaught exception. In this situation, no easily parsed error is found so returning an error string is the next best thing.